examples/flutter_gallery/lib/demo/cupertino/cupertino_picker_demo.dart

```diff
@@ -91,12 +90,10 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
                 scrollController: scrollController,
                 itemExtent: _kPickerItemHeight,
                 backgroundColor: CupertinoColors.white,
-                onSelectedItemChanged: (int index) {
-                  setState(() => _selectedColorIndex = index);
-                },
+                onSelectedItemChanged: (int index) { setState(() => _selectedColorIndex = index); },
```

Comment:
i think this one is less readable than the original

Reply:
With `setState(() { _selectedColorIndex = index; });` it will be expanded.